lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -606,6 +606,7 @@
       filename:
         description:
           - Name of the file containing the secret. Defaults to the I(secret_name) if not specified.
+          - Corresponds to the C(target) option of C(docker service create --secret).
```

Comment:
I think the following would be better:
```suggestion
          - Corresponds to the C(target) key of C(docker service create --secret).
```
I would say `target` isn't an option, but the key of a key-value pair specified to `--secret`. (https://docs.docker.com/engine/reference/commandline/service_create/#create-a-service-with-secrets)

Does that make sense?

Reply:
It does, I was unsure on the nomenclature used in the docs!

Reply:
I'm not sure whether that's the same nomenclature the Docker docs are using, but I think it is good enough for us :)